vacation/vacation.go

```diff
@@ -479,7 +479,7 @@ func (z ZipArchive) Decompress(destination string) error {
 // directory
 func checkExtractPath(filePath string, destination string) error {
 	destpath := filepath.Join(destination, filePath)
-	if !strings.HasPrefix(destpath, filepath.Clean(destination)+string(os.PathSeparator)) {
+	if !strings.HasPrefix(destpath, filepath.Clean(destination)+"/") {
```

Comment:
I needed one more change to get the comparison right:
```golang
func checkExtractPath(tarFilePath string, destination string) error {
	osPath := filepath.FromSlash(tarFilePath)
	destpath := filepath.Join(destination, osPath)
	if !strings.HasPrefix(destpath, filepath.Clean(destination)+string(os.PathSeparator)) {
		return fmt.Errorf("illegal file path %q: the file path does not occur within the destination directory", tarFilePath)
	}
	return nil
}
```	

Full commit (with yours as a parent) is here: https://github.com/micahyoung/packit/commit/869a1e0b98bc4a2b0dd23d3bdb4d2af3b9775f5a

Reply:
@micahyoung looks good to me! it has been added and is ready for testings.